common/lib/xmodule/xmodule/tests/test_library_content.py

```diff
@@ -74,18 +74,14 @@ class TestLibraryContentExportImport(LibraryContentTest):
     """
     Export and import tests for LibraryContentBlock
     """
+    def setUp(self):
+        super().setUp()
```

Comment:
You don't need to define setUp if it only calls super().setUp().  The parent's setUp will be inherited.

Reply:
@nedbat, it looks a bit confusing in the diff, but both `super().setUp()` and `self.lc_block.refresh_children()` have the same indentation level.